test/alexa/AlexaEvent.spec.ts

```diff
@@ -69,3 +73,57 @@ describe("AlexaEvent", () => {
     expect(alexaEvent.intent.params).to.be.ok;
   });
 });
+
+describe("LoginWithAmazon", () => {
+  const rb = new AlexaRequestBuilder();
+  let voxaApp: VoxaApp;
+  let alexaSkill: AlexaPlatform;
+
+  beforeEach(() => {
+    voxaApp = new VoxaApp({ views, variables });
+    alexaSkill = new AlexaPlatform(voxaApp);
+  });
+
+  afterEach(() => {
+    nock.cleanAll();
+  });
+
+  it("should validate user information", async () => {
+    const lwaResult: IVoxaAlexaUserProfile = {
+      email: "johndoe@example.com",
+      name: "John Doe",
+      userId: "amzn1.account.K2LI23KL2LK2",
+      zipCode: "12345",
+    };
+
+    nock("https://api.amazon.com")
+      .get("/user/profile?access_token=accessToken")
+      .reply(200, {
+        email: "johndoe@example.com",
+        name: "John Doe",
+        postal_code: "12345",
+        user_id: "amzn1.account.K2LI23KL2LK2",
+      });
+
+    const rawEvent = rb.getLaunchRequest();
+    _.set(rawEvent, "session.user.accessToken", "accessToken");
+
+    const alexaEvent = new AlexaEvent(rawEvent) as VoxaEvent;
+    alexaEvent.platform = alexaSkill;
+
+    const userDetails = await alexaEvent.getUserInformation();
+    expect(userDetails).to.deep.equal(lwaResult);
+  });
+
+  it("should throw an error when accessToken is empty", async () => {
+    const rawEvent = rb.getLaunchRequest();
+    const alexaEvent = new AlexaEvent(rawEvent) as VoxaEvent;
+    alexaEvent.platform = alexaSkill;
+
+    try {
```

Comment:
maybe also add something to actually make sure the exception is thrown? right now we're only testing that if the exception is thrown the message is what we expect


```suggestion
    let exceptionWasThrown = false;
    try {
      await alexaEvent.getUserInformation();
    } catch (err) {
      exceptionWasThrown = true;
      expect(err.message).to.equal("this.user.accessToken is empty");
    }
    expect(exceptionWasThrown).to.be.true;
```

Reply:
@armonge done